NETCORE/src/Microsoft.ApplicationInsights.AspNetCore/Logging/Implementation/ApplicationInsightsLoggerProvider.cs

```diff
@@ -9,7 +9,7 @@
     /// <summary>
     /// <see cref="ILoggerProvider"/> implementation that creates returns instances of <see cref="ApplicationInsightsLogger"/>.
     /// </summary>
-#if !NETSTANDARD2_0
+#if !NETSTANDARD2_0 || !NET461
```

Comment:
Should this be `&&`  (and update the comment on the following line to match)

Reply:
Thanks for reviewing. I will update both!